src/app/shared/components/capture-receipt/receipt-preview/receipt-preview.component.ts

```
@@ -200,43 +236,63 @@ export class ReceiptPreviewComponent implements OnInit {
       },
       cssClass: 'pop-up-in-center',
     });
-
     await deletePopOver.present();
-
-    const { data } = await deletePopOver.onWillDismiss();
-
+    const { data }: { data?: { action?: string } } = await deletePopOver.onWillDismiss();
     if (data && data.action) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)